onyo/lib/filters.py

```diff
@@ -49,7 +49,7 @@ def _re_match(text: str, r: str) -> bool:
             return False
 
     def match(self, asset: dict) -> bool:
-        """match self on a dictionary"""
+        r"""match self on a dictionary"""
```

Comment:
Trailing `.` missing.

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.